admin_guide/cluster_management.rst

```diff
@@ -193,6 +193,54 @@ The coordinator node needs to know roles' passwords in order to communicate with
 
 Sometimes workers need to connect to one another, such as during :ref:`repartition joins <repartition_joins>`. Thus each worker node requires a copy of the .pgpass file as well.
 
+.. _phone_home:
+
+Checks For Updates and Cluster Statistics
+#########################################
+
+Unless you opt out, Citus checks if there is a newer version of itself during installation and every twenty-four hours thereafter. If a new version is available, Citus emits a notice to the database logs.
+
+During the check for updates, Citus also sends general information about the running cluster to Citus Data company servers. This helps us understand how Citus is commonly used and thereby improve the product. As explained below, the reporting is opt-out and does **not** contain personally identifying information about schemas, tables, queries, or data.
+
+What we Collect
+---------------
+
+* Citus checks if there is a newer version of itself, and if so emits a notice to the database logs.
+* Citus collects and sends these statistics about your cluster:
+   * Randomly generated cluster identifier
+   * Number of workers
+   * OS version and hardware type (output of ``uname -psr`` command)
+   * Number of tables, rounded to a power of two
+   * Total size of shards, rounded to a power of two
+   * Whether Citus is running in Docker or natively
+
+Because Citus is an open-source PostgreSQL extension, the statistics reporting code is available for you to audit. See :code:`CitusMaintenanceDaemonMain` in `maintenanced.c <https://github.com/citusdata/citus/blob/master/src/backend/distributed/utils/maintenanced.c>`_ and the functions in `statistics_collection.c <https://github.com/citusdata/citus/blob/master/src/backend/distributed/utils/statistics_collection.c>`_.
```

Comment:
Any reason to include the link to maintenance daemon? Since that seems mostly just basic background worker stuff. Also, if we want to be more specific, we can provide the exact link: https://github.com/citusdata/citus/blob/master/src/backend/distributed/utils/statistics_collection.c#L85. But the file is fine as well.

Reply:
I included that link because I thought it would show when Citus is attempting to call home, but I can remove the link for simplicity.

Also I didn't link to a specific line number because it seemed likely to change and break the link.